src/API/Google/AdsAssetGroupAsset.php

```diff
@@ -44,6 +48,14 @@ class AdsAssetGroupAsset implements OptionsAwareInterface {
 	 */
```

Comment:
In this file

`use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\AdsAsset;`
Is not being used.